packages/sdks/snippets/hydrogen/app/components/AnnouncementBarPage.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Quickstart snippet
+ * snippets/hydrogen/app/components/AnnouncementBarPage.tsx
+ */
+
+import {
+  Content,
+  fetchOneEntry,
+  getBuilderSearchParams,
+} from '@builder.io/sdk-react';
+import type {LoaderFunction} from '@remix-run/node';
+import {useLoaderData} from '@remix-run/react';
+import {useNonce} from '@shopify/hydrogen';
+
+const BUILDER_API_KEY = 'ee9f13b4981e489a9a1209887695ef2b';
+
+export const announcementsLoader: LoaderFunction = async ({
+  params,
+  request,
+}) => {
+  try {
+    const pathname = `/announcements/${params['*'] || ''}`;
+    const url = new URL(request.url);
+
+    const model = 'announcement-bar';
```

Comment:
this model can be moved out of the loader in the module scope (next to the API key) so its not passed around unnecessarily